tests/wpt/meta/html/dom/reflection-forms.html.ini

```diff
@@ -3380,132 +3380,24 @@
   [meter.tabIndex: IDL set to -2147483648]
     expected: FAIL
 
-  [meter.value: typeof IDL attribute]
-    expected: FAIL
-
-  [meter.value: IDL set to -10000000000]
-    expected: FAIL
-
-  [meter.value: IDL set to -1]
-    expected: FAIL
-
   [meter.value: IDL set to -0]
     expected: FAIL
```

Comment:
Apparently some setters do not work correctly (maybe wrong parsing?).

Reply:
Sorry for not responding to this comment yet but I went down a rabbit hole, I switched the setter implementations to `parse_floating_point_number` before setting the value, but the issue persisted.

I'm really unsure if this is something in `parse_floating_point_number` or not :/

[Spidermonkey's](https://searchfox.org/mozilla-central/source/dom/base/ResponsiveImageSelector.cpp#36) `parseFloat(-0.0)` returns `0`. But our implementation returns -0.0 - do you think this might be a factor or am I barking up the wring tree?

Thank you!

Reply:
I think that our parse_floating_point_number is wrong, but we can leave this for follow up. In firefox the corresponding function is actually https://searchfox.org/mozilla-central/source/dom/base/nsAttrValue.cpp#1892 (you can see it's only used on progress and meter element just like ours).